lab-raziyeh/server.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+//node modules
+const http = require('http');
+const url = require('url');
+const queryString = require('querystring');
+
+//npm modules
+const cowsay = require('cowsay');
+
+//app modules
+const parseBody = require('./lib/parse-body.js');
+
+//module constants
+const PORT = process.env.PORT || 3000;
+
+
+//module logic
+const server = http.createServer(function(req, res) {
+  req.url = url.parse(req.url);
+  req.url.query = queryString.parse(req.url.query);
+
+//Method : GET
+  if (req.method === 'GET') {
+    if(req.url.pathname === '/') {
+      res.writeHead( 200, { 'Content-Type': 'text/plain'});
```

Comment:
@Raziyehbazargan While it may seem odd at first, best practice is to break this statement onto new lines.  For example:

```
res.writeHead(200, {
  'Content-Type': 'text/plain'
});
```